### PR TITLE
feat: added titles to post-meta data

### PR DESCRIPTION
**Pull request by tinect**
I'd assume there are people not understanding icons, so I added titles.
What do you think about it?

### CODE REVIEW

**Comment by razonyang** [comment]
LGTM, how about replacing the prefix `title_` with `post_`? I think the latter makes more sense than title, since they are belonged to posts.

File: i18n/de.toml [hunk 1]
```diff
@@ -116,3 +116,12 @@ other = "Themen"
...
 other = "WeChat"
+
+[title_created_on]
+other = "erstellt am"
+
+[title_updated_on]
+other = "aktualisiert am"
+
+[title_reading_time]
+other = "Lesezeit"
```
Comment (razonyang) [inline i18n/de.toml:127]: I prefer to place keys in alphabetical order(for maintaining reason).

File: i18n/en.toml [hunk 2]
```diff
@@ -115,3 +115,12 @@ other = "Tags"
...
 other = "WeChat"
+
+[title_created_on]
+other = "created on"
+
+[title_updated_on]
+other = "updated on"
+
+[title_reading_time]
+other = "reading time"
```
Comment (razonyang) [inline i18n/en.toml:126]: Should be ended with newline.

**Comment by tinect** [comment]
> LGTM, how about replacing the prefix `title_` with `post_`? I think the latter makes more sense than title, since they belong to posts.

Oh, yeah. Correct :-)
I've updated the PR.

**Review by razonyang — APPROVED** [review]

**Comment by razonyang** [comment]
Thanks for your contributions.

**Pull request merged**